src/index.css

```diff
@@ -7,9 +7,18 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: var(--background-color);
+  color: var(--foreground-color);
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.dark--mode {
+  --background-color: rgb(41, 39, 39);
+  --foreground-color: rgb(143, 143, 143);
+  --logo-toggle: whitesmoke;
+  --container-color: #444;
+}
```

Comment:
minor suggestion to keep consitency with color values. Rest whole code looks clean and consistent. UI is good